airflow/jobs/backfill_job_runner.py

```diff
@@ -631,6 +631,20 @@ def _per_task_process(key, ti: TaskInstance, session):
                                     "Not scheduling since Task concurrency limit is reached."
                                 )
 
+                        if task.max_active_tis_per_dagrun:
```

Comment:
Same for checks against this, I feel `is not None` is more logical? (Although the difference is really minimal in practice.)

Reply:
Same for `if task.max_active_tis_per_dag:`, I fixed them both